meson.build

```diff
@@ -69,6 +70,10 @@ if enable_breadcrumbs
   add_project_arguments('-DVKD3D_ENABLE_BREADCRUMBS', language : 'c')
 endif
 
+if enable_tracy
+  add_project_arguments('-DTRACY_ENABLE', language : 'c')
```

Comment:
enable_tracy should also imply enable_profiling.

Reply:
Will update.